### PR TITLE
chore: update codegen

### PR DESCRIPTION
**Pull request by lundeeell**
**Ticket:** [#93](https://trello.com/c/9DhBs4qa/98-update-deprecated-graphql-code-generator)

**Intent:**
Update deprecated codegen packages

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,11 +76,6 @@
     "eslint": "^5.14.1",
     "eslint-config-prettier": "^4.0.0",
     "got": "^9.6.0",
-    "graphql-code-generator": "^0.17.0",
-    "graphql-codegen-introspection": "^0.17.0",
-    "graphql-codegen-typescript-common": "^0.17.0",
-    "graphql-codegen-typescript-resolvers": "^0.17.0",
-    "graphql-codegen-typescript-server": "^0.17.0",
```
Comment (Graborg) [inline package.json]: Was typescript server not needed any more?

**Comment by Graborg** [comment]
Checks out!

File: package.json (hunk 1 above)
Comment (lundeeell) [reply package.json]: No, new packages! :)

**Pull request merged**